UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Program.cs

```diff
@@ -172,5 +178,32 @@ private static HostBuilder BuildHostConfiguration()
 
             return hostBuilder;
         }
+
+        private static IAsyncPolicy<HttpResponseMessage> GetRetryPolicy(Microsoft.Extensions.Logging.ILogger logger, string requestType, EventIds eventId)
```

Comment:
This looks identical to the policy in the `UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Startup.cs`, is there a good common place we can define the policy and reference here rather than defining the retry policy twice?

Reply:
Yes. Right moved to common helper class referred from there